src/AnchoredOverlay/AnchoredOverlay.tsx

```diff
@@ -168,6 +169,7 @@ export const AnchoredOverlay: React.FC<AnchoredOverlayProps> = ({
           top={position?.top || 0}
           left={position?.left || 0}
           anchorSide={position?.anchorSide}
+          {...(portalContainerName ? {portalContainerName} : {})}
           {...overlayProps}
```

Comment:
We already have an option to pass any `overlayProps` through the Anchored Overlays, which seems like an OK path for this option as well.  It should be rarely used, kind of like height/width which use this other approach.  Should significantly cut down on the surface area of this PR as well